### DISCUSSION TITLE
LabelFrame

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#658] [Ideas]
@TomSchimansky are there any plans for a CTkLabelFrame?

Thanks

**Comment by cavalb** [comment]
I agree
Is a important widget for me too.

**Comment by UX0l0l** [comment]
Same here. I'm trying to make an app and need a CTk native LabelFrame. Also, by any chance does anyone know how to color the text bg of the LabelFrame?
